### ISSUE TITLE
App is crashing on 3.0 using Toast on web context

### ISSUE THREAD

**Issue by ScreamZ** [driftyco/ionic-native#1346]
Hi,

in order to display my toast I must call the subscribe method. But using this behavior in the web browser while "ionic serve" cause the app to crash with message : 

```
plugin.js:167 Unable to clear the previous observable watch for Toast showShortBottom
plugin.js:168 TypeError: Cannot read property 'hide' of null(…)(anonymous function) @ plugin.js:168tryCatcher @ tryCatch.js:6Subscription.unsubscribe @ Subscription.js:68Subscription.add @ Subscription.js:134Observable.subscribe @ Observable.js:45HomePage.addPlayer @ home.ts:39(anonymous function) @ HomePage.html:29handleEvent @ core.es5.js:11805callWithDebugContext @ core.es5.js:13013debugHandleEvent @ core.es5.js:12601dispatchEvent @ core.es5.js:8780(anonymous function) @ core.es5.js:9370(anonymous function) @ platform-browser.es5.js:2683t.invokeTask @ polyfills.js:3onInvokeTask @ core.es5.js:4116t.invokeTask @ polyfills.js:3n.runTask @ polyfills.js:3invoke @ polyfills.js:3
HomePage.html:29 ERROR Error: cordova_not_available(…)
```

Any way to show a warning like these : util.js:60Native: tried calling StatusBar.styleDefault, but Cordova is not available. Make sure to include cordova.js or run in a device/simulator

Best regards

**Comment by ihadeed** [comment]
More reasons to review #395

**Comment by ihadeed** [comment]
Fixed in 3.9.1, please upgrade `@ionic-native/core` to `3.9.1` and the issue should be fixed.
